devito/symbolics/inspection.py

```diff
@@ -79,7 +79,7 @@ def estimate_cost(exprs, estimate=False):
             * Powers with integer exponent `n>0` count as n-1 ops (as if
               it were a chain of multiplications).
     """
-    trascendentals_cost = {sin: 50, cos: 50, exp: 50, log: 50}
+    trascendentals_cost = {sin: 100, cos: 100, exp: 100, log: 100}
```

Comment:
Why the increase?

Reply:
as the PR title says, I've tweaked the cost model for aliases. Basically now I'm not lifting expressions (and therefore I'm not allocating tensor temporaries, thus increasing the working set size) if the trade-off with the saved number of operations isn't favorable. I've tweaked the weights and the CIRE algorithms to improve the cost model